package.json

```diff
@@ -93,6 +93,7 @@
     "@types/blueimp-md5": "^2.18.0",
     "@types/clean-css": "^4.2.2",
     "@types/jquery": "^3.3.31",
+    "@types/jquery.validation": "^1.16.7",
```

Comment:
This will require a minor provision bump.

Reply:
fixed!